apertium-eng.eng.dix

```diff
@@ -15938,7 +15938,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <e lm="liveliness">      <i>liveliness</i><par n="access__n"/></e>
 <e lm="liveness">        <i>liveness</i><par n="access__n"/></e>
 <e lm="liver">           <i>liver</i><par n="house__n"/></e>
-<e lm="livermorium">           <i>livermorium</i><par n="house__n"/></e>
+<e lm="livermorium"> <i>livermorium</i><par n="house__n"/></e>
```

Comment:
This might be a tab-indenting vs space-indenting issue, but many of the words are still not aligned

Reply:
I don't see anything on my side. How do I fix that? I'm using notepad right now.

Reply:
I think notepad uses tabs to indent while Apertium uses spaces. Could you try to use spaces and see how it looks?